### PR TITLE
poc-refactor-ssr/csr-app-rendering

### PR DESCRIPTION
**Pull request by jroebu14**
Resolves #NUMBER

**Overall change:**
_A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added labels to this PR for the relevant pod(s) affected by these changes
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/App/App.jsx [hunk 1]
```diff
@@ -1,124 +1,66 @@
...
-          assetUri: nextAssetUri,
-          isAmp: nextIsAmp,
-          pageType: route.pageType,
-          loading: true,
-          error: null,
-          errorCode: null,
-          timeOnServer: null,
-        });
-      });
+    useEffect(() => {
```
Comment (chris-hinds) [inline src/app/containers/App/App.jsx:40]: Does this remove the loading state?

Comment (jroebu14) [reply src/app/containers/App/App.jsx:40]: it does. was anything using it? can be added back in

Comment (chris-hinds) [reply src/app/containers/App/App.jsx:40]: there was intention to have an actual loading component/ui, there was some work done a little while ago as well that suspends the page rendering for a small amount of time i.e. if the data fetch is super fast, we don't show a loading page. if the data fetch is slow we transition to a loading page.

However this really only matters when client side rendering.

Comment (jroebu14) [reply src/app/containers/App/App.jsx:40]: ah ok I'll add that stuff back in

**Pull request closed without merging**